tests/test_phishingpage.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python2
+
+import unittest
+import phishingpage
+import os
+
+
+class TestUrlChecker(unittest.TestCase):
```

Comment:
A false URL won't resolve at all. It's something like "www.1dont3x1st22.com". 

A 404 error code on the other hand, is a non-existent page in a valid domain, like "imgur.com/aZasdfwefuijasdf".